wizard.py

```
@@ -165,213 +209,268 @@ def select_services(transcription_provider=None):
-        console.print(f"[yellow]   Check that the service directory exists: {service['path']}[/yellow]")
-        console.print(f"[yellow]   And that 'uv' is installed and on your PATH[/yellow]")
+        console.print(
+            f"[yellow]   Check that the service directory exists: {service['path']}[/yellow]"
+        )
+        console.print(
+            f"[yellow]   And that 'uv' is installed and on your PATH[/yellow]"
+        )
         return False
     except subprocess.TimeoutExpired as e:
         console.print(f"❌ {service_name} setup timed out after {e.timeout}s")
         console.print(f"[yellow]   Configuration may be partially written.[/yellow]")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)